### DISCUSSION TITLE
[WASM] Link to index.html template for .NET7 broken?

### DISCUSSION THREAD

**Discussion by DierkDroth** [unoplatform/uno#10415] [Q&A] [answered]
While trying to get UNO WASM working on .NET7 I was suspicious that I might be using the wrong index.html and went [here ](https://platform.uno/docs/articles/external/uno.wasm.bootstrap/doc/features-additional-files.html)to pull the correct template.

However the link for the .NET7 template appears to broken.

Where could I find the correct template for .NET7?

**Comment by jeromelaban** [comment, marked as answer]
That's correct. Following the previous links, you can extrapolate the path to https://github.com/unoplatform/Uno.Wasm.Bootstrap/blob/release/stable/7.0/src/Uno.Wasm.Bootstrap/Templates/index.html.

**Reply by DierkDroth** [reply]
Thanks @jeromelaban. This got me one step farther ...
